### PR TITLE
Coco/custom404

### PR DESCRIPTION
**Pull request by CocoisBuggy**
Nice simple 404 page that fulfils the base spec.

At the very least, it's not a blank page with invisible error message (The current solution in place)

Bit of fuss with git reversion, looks like 404.js did not get properly migrated. If that history is important, I can go back and do the steps in the correct order.

### CODE REVIEW

File: src/pages/404.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Header from '../components/Header'
+
+function NotFoundPage (): JSX.Element {
+  return (
+    <>
+      <div className='absolute w-full h-screen bg-gray-50 overflow-hidden p-2'>
```
Comment (vnugent) [inline src/pages/404.tsx]: I think `h-screen` pushes the page footer out of the viewport.

File: src/pages/404.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+              <div className='bg-yellow-100 italic mb-2'>
+                404
+              </div>
+            </div>
+
+            <div className='bg-white text-4xl md:text-7xl tracking-wide font-bold px-2 py-1'>
+              Page not found
+            </div>
+
+            <div className='mt-2 bg-white'>
```
Comment (vnugent) [inline src/pages/404.tsx]: can we leave out the background (bg-white) to let the div inherit whatever is set higher up?  

Screen Shot 2022-05-15 at 11 11 55 AM

**Review by vnugent — CHANGES_REQUESTED** [review]

File: src/pages/404.tsx (hunk 1 above)
Comment (CocoisBuggy) [reply src/pages/404.tsx]: Footer was left out, I have changed from manual usage of header to <Layout />

File: src/pages/404.tsx (hunk 2 above)
Comment (CocoisBuggy) [reply src/pages/404.tsx]: Sure thing

**Pull request merged**